Master

cthanos opened the pull request:
-Login.jsp
-User Authentication(UserDAO,User,UserAuthentication)
-JS/JQUERY DOM manipulation

cthanos committed:
Merge remote-tracking branch 'dalexo/master'

cthanos committed:
Merge remote-tracking branch 'dalexo/master'

cthanos committed:
Implementation of:

-Login.jsp
-UserAuthentication(UserDAO.java,User.java,UserAuthentication.java)
-JS DOM manipulation

cthanos committed:
Implementation of:
-Login.jsp
-User Authentication(UserDAO,User,UserAuthentication)
-JS/JQUERY DOM manipulation

gmetal commented on AcademicManagerV1/src/com/academic/db/dao/UserDAO.java:
```diff
@@ -0,0 +1,89 @@
+	@Override
+	public void close() {
+		// TODO Auto-generated method stub
+		
```
You must implement this method.

gmetal commented on AcademicManagerV1/src/com/academic/db/dao/UserDAO.java:
```diff
@@ -0,0 +1,89 @@
+	@Override
+	public int countAll() {
+		// TODO Auto-generated method stub
+		return 0;
```
You should implement this method.

gmetal commented on AcademicManagerV1/src/com/academic/db/dao/UserDAO.java:
```diff
@@ -0,0 +1,89 @@
+	@Override
+	public User get(int id) {
+		// TODO Auto-generated method stub
+		return null;
```
You should implement this method.

gmetal reviewed (commented):
Please remove the node_modules directory and only install the jquery.min.js.

cthanos committed:
Commit Description

- Remove ’node_modules’ folder
- Implement getById(), close() & countAll() methods on UserDAO

gmetal commented on AcademicManagerV1/WebContent/js/loginFailed.js:
```diff
@@ -0,0 +1,16 @@
+
+
+
+
```
Please remove all these empty lines

gmetal commented on AcademicManagerV1/src/com/academic/db/dao/UserDAO.java:
```diff
@@ -0,0 +1,128 @@
+
+	@Override
+	public List<User> getAll() {
+		// TODO Auto-generated method stub
```
Remove TODO

cthanos committed:
Commit Description

Security check on homepage

gmetal commented on AcademicManagerV1/src/com/academic/servlets/UserAuthentication.java:
```diff
@@ -0,0 +1,91 @@
+	 */
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+		// TODO Auto-generated method stub
```
You should call doPost rather than provide the default implementation

gmetal commented on AcademicManagerV1/src/com/academic/db/dao/UserDAO.java:
```diff
@@ -0,0 +1,128 @@
+
+	public UserDAO(Connection conn) throws SQLException {
+		super(conn);
+		// TODO Auto-generated constructor stub
```
Remove TODO

gmetal commented on AcademicManagerV1/src/com/academic/servlets/UserAuthentication.java:
```diff
@@ -0,0 +1,91 @@
+	 */
+	public UserAuthentication() {
+		super();
+		// TODO Auto-generated constructor stub
```
Remove TODO

cthanos committed:
Commit Description

Fix some issues

cthanos committed:
Commit Description

Logout implementation..

gmetal commented:
LGTM
